src/alerts/dataclasses.py

```diff
@@ -0,0 +1,31 @@
+from dataclasses import dataclass
+from typing import List
+
+
+@dataclass
+class LimitInterval:
+    def __init__(self, lb: float, ub: float):
+        self.lower_bound = lb
+        self.upper_bound = ub
```

Comment:
Define it better in that way, init method is used to add extra logic
```suggestion
class LimitInterval:
    lower_bound: float
    upper_bound float
```